### PR TITLE
[bitnami/harbor] Update chart requirements and fix default password in README

### PR DESCRIPTION
**Pull request by marcosbc**
**Description of the change**

Update PostgreSQL and Redis chart dependency versions. This also fixes an error in the README regarding the admin password.

**Benefits**

Use PostgreSQL and Redis containers without security issues.

**Possible drawbacks**

Some parts may not work properly as Harbor officially still includes PostgreSQL 9.6.14 and Redis 4.0.10. However I did a thorough test and it worked perfectly for me.

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/harbor/requirements.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
-  version: 4.x.x
+  version: 5.x.x
```
Comment (carrodher) [inline bitnami/harbor/requirements.yaml]: A new PostgreSQL major is coming, it should be merged today: https://github.com/helm/charts/pull/15643

**Comment by marcosbc** [comment]
Done, I basically copied the existing section apart from a couple of modifications, as in this case PostgreSQL is a dependency of the chart rather than a bundled service.

File: bitnami/harbor/README.md [hunk 2]
```diff
@@ -347,3 +347,36 @@ This chart includes a `values-production.yaml` file where you can find some para
 It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
 
 Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+## Upgrade
+
+## 2.0.0
+
+In this version, this **chart depends on PostgreSQL 11 instead of PostgreSQL 10**. You can find the main difference and notable changes in the following links: [https://www.postgresql.org/about/news/1894/](https://www.postgresql.org/about/news/1894/) and [https://www.postgresql.org/about/featurematrix/](https://www.postgresql.org/about/featurematrix/).
```
Comment (tompizmor) [inline bitnami/harbor/README.md]: I would also mention that Redis goes from version 4.0 to 5.0.

Comment (marcosbc) [reply bitnami/harbor/README.md]: Good point! Updated

**Pull request merged**